models/renalStructures_CECT_segmentation/configs/metadata.json

```diff
@@ -0,0 +1,84 @@
+{
+    "schema": "https://github.com/Project-MONAI/MONAI-extra-test-data/releases/download/0.8.1/meta_schema_20220324.json",
+    "version": "0.1.0",
+    "changelog": {
+        "0.1.0": "complete the model package"
+    },
+    "monai_version": "1.2.0",
+    "pytorch_version": "1.13.1",
+    "numpy_version": "1.24.3",
+    "optional_packages_version": {
+        "nibabel": "5.1.0",
+        "pytorch-ignite": "0.4.11",
+        "einops": "0.6.1",
+        "fire": "0.5.0",
+        "torchvision": "0.15.2"
```

Comment:
The CI tests failed because of lib conflicts. Could you double confirm the package versions?
I checked the torchvision and `0.15.2` requires torch 2.0
https://github.com/pytorch/vision#installation

Reply:
Yes, you are right. I will change torchvision to match pytorch (I think I had torch 2.0 installed and downgraded it, but without downgrading torchvision).